docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -105,6 +105,52 @@ include::../../../../extensions/agroal/deployment/src/test/resources/application
 
 There are other configuration options, detailed below.
 
+=== JDBC URL configurations
+
+As noted above, {project-name} currently supports four databases.
```

Comment:
I would avoid saying that because I'm pretty sure we will forget to update it, next time we add a supported database.